terraform/environments/analytical-platform-ingestion/transfer-family/modules/transfer-family/user-with-egress/variables.tf

```diff
@@ -0,0 +1,39 @@
+variable "name" {
+  type = string
+}
```

Comment:
if possible, should we added a regex condition to this to enforce lower cased kebab users?